reflex/components/core/cond.py

```diff
@@ -59,14 +62,17 @@ def create(
         )
 
     def _get_props_imports(self):
-        """Get the imports needed for components props.
+        """Get the imports needed for component's props.
 
         Returns:
-            The  imports for the components props of the component.
+            The imports for the component's props of the component.
         """
         return []
 
     def _render(self) -> Tag:
+        self.comp1.apply_theme(self.theme)  # type: ignore
+        self.comp2.apply_theme(self.theme)  # type: ignore
```

Comment:
shouldn't this happen inside `_apply_theme`? then you wouldn't need to have a `theme` prop on `Cond` or `Match`

Reply:
Agreed, I suppose for cond this special case isn't really needed. Whereas in the foreach component we generate a new component from the tag.